diff --git a/base/BUILD.gn b/base/BUILD.gn
index 4f34b199d7839c2c560d669a221ec0c9c2005a15..e99041ca5415654adfd789dcd40d233d9af38606 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4456,6 +4456,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4930,6 +4931,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   # Most cached_flags code is in components/cached_flags. These classes are
