diff --git a/base/trace_event/builtin_categories.h b/base/trace_event/builtin_categories.h
index 435735a272bea0e67346ee6388e237e190854c8d..8d77d5ae08928a4640f5c328316c16071ff0fcb1 100644
--- a/base/trace_event/builtin_categories.h
+++ b/base/trace_event/builtin_categories.h
@@ -205,6 +205,7 @@
   X("webrtc")                                                            \
   X("webrtc_stats")                                                      \
   X("xr")                                                                \
+  BRAVE_INTERNAL_TRACE_LIST_BUILTIN_CATEGORIES(X) /*see chromium_src*/   \
   X(TRACE_DISABLED_BY_DEFAULT("android_view_hierarchy"))                 \
   X(TRACE_DISABLED_BY_DEFAULT("animation-worklet"))                      \
   X(TRACE_DISABLED_BY_DEFAULT("audio"))                                  \
