diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index a5166b625e0e535ad7a0a84d36122399733903fc..c6a9788360c4863e13e4c32b9c1f3866a19c6821 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -42,3 +42,4 @@ build_dotfile_settings = {
     "//build/util/branding.gni",
   ]
 }
+import("//brave/build/dotfile_settings.gni") build_dotfile_settings.exec_script_whitelist += brave_exec_script_whitelist
