diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 5ef365b3168b85757a697497abed708100ae28e6..b5a6b06b4556ef44b7fea5a6bc3776056d2afc24 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4197,6 +4197,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
