diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
index 89d7870cf46970f46269e558e25e5f98abb12924..4354a9dd139dfd9b91b8a394996baa5738531585 100644
--- a/chrome/browser/hub/internal/BUILD.gn
+++ b/chrome/browser/hub/internal/BUILD.gn
@@ -46,6 +46,7 @@ android_library("java") {
     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactory.java",
     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactoryImpl.java",
   ]
+  sources += brave_browser_hub_internal_java_sources
   deps = [
     ":java_resources",
     "//base:base_java",
