diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index b84f6dad25732df847cf527ea277fa457b26dddd..7ec56ca0f7662358be6f460c7fec4b97714d540a 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -601,6 +601,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
