diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index d26f6e2bdefd8ce5a3871b1002c682def17434a3..a6558e6a6597bb28101643c55c68d66210d6a5cb 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -637,6 +637,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
       return nullptr;
     }
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/40136163): Remove this code after we integrate with intent
@@ -1031,6 +1032,7 @@ bool IsURLAllowedInIncognito(const GURL& url) {
     }
     return stripped_url.is_valid() && IsURLAllowedInIncognito(stripped_url);
   }
+  BRAVE_IS_URL_ALLOWED_IN_INCOGNITO
 
   return IsHostAllowedInIncognito(url);
 }
