diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 4b26d2090f878a4a6f522c91c7315572364fba05..653f5fa26b07b2a7ba000f90620f3b1b78e21783 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -692,6 +692,7 @@ class TabStripModel : public TabGroupController {
   tabs::TabModel* GetTabForWebContents(
       const content::WebContents* contents) const;
 
+  TAB_STRIP_MODEL_H_
  private:
   FRIEND_TEST_ALL_PREFIXES(TabStripModelTest, GetIndicesClosedByCommand);
 
