diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 13f2a16eea57739608e24c1c8ea806794f73255d..e63f4dbd1372acaef7b3777ee645dce918a3b8de 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1477,6 +1477,7 @@ source_set("extensions") {
       ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions) {
