diff --git a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
index f840d8339df4780e49dd1bb6f3fda882a8190da1..4094ed5e8eecb52fc26a61eb6c574b51ae1dccad 100644
--- a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
+++ b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
@@ -298,7 +298,7 @@ std::unique_ptr<VideoOverlayWindowViews> VideoOverlayWindowViews::Create(
   // constructor. It's important that the constructor be private, because it
   // doesn't initialize the object fully.
   auto overlay_window =
-      base::WrapUnique(new VideoOverlayWindowViews(controller));
+      base::WrapUnique(new BraveVideoOverlayWindowViews(controller));
 
   // The 2024 updated controls use dark mode colors.
   if (Use2024UI()) {
@@ -760,6 +760,7 @@ void VideoOverlayWindowViews::UpdateMaxSize(const gfx::Rect& work_area) {
 
   max_size_ = new_max_size;
 
+  BRAVE_UPDATE_MAX_SIZE
   if (!native_widget())
     return;
 
