diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index b7ee79c7e28b3fe01bbcd3c23b75b6686a12a942..40083e06973ce33945e7e7c42fd579b769863547 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -611,6 +611,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
