diff --git a/components/sync/engine/data_type_worker.h b/components/sync/engine/data_type_worker.h
index 642a470663a245da245ecf4172b0e4c65e2dee3e..568d83fc1303382769fd4aeabd919e4bd22365a1 100644
--- a/components/sync/engine/data_type_worker.h
+++ b/components/sync/engine/data_type_worker.h
@@ -213,6 +213,7 @@ class DataTypeWorker : public UpdateHandler,
 
   static constexpr size_t kMaxPendingInvalidations = 10u;
 
+ BRAVE_MODEL_TYPE_WORKER_H_
  private:
   struct UnknownEncryptionKeyInfo {
     // Not increased if the cryptographer knows it's in a pending state
