diff --git a/components/policy/resources/webui/policy_base.ts b/components/policy/resources/webui/policy_base.ts
index 27109a436a02b8a56cd9335df95d374d4f0d9175..6bd2771dee3ba0093781d91c3b768807a8d0dcff 100644
--- a/components/policy/resources/webui/policy_base.ts
+++ b/components/policy/resources/webui/policy_base.ts
@@ -209,7 +209,7 @@ export class Page {
                       policyNames['precedence']?.policyNames,
                     ].includes(knownPolicyNames) &&
                             knownPolicyNamesSet.has(name) ?
-                        `https://chromeenterprise.google/policies/?policy=${
+                        `https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy?policy=${
                             name}` :
                         undefined,
                   },
