diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
index 8498ce3a67efcb453889ac8c622771f0bf4b1535..d142fc9fbb91db326489b0d82f5df6b473cd748d 100644
--- a/components/content_settings/core/browser/content_settings_utils.cc
+++ b/components/content_settings/core/browser/content_settings_utils.cc
@@ -171,6 +171,7 @@ bool CanTrackLastVisit(ContentSettingsType type) {
   if (type == ContentSettingsType::PROTOCOL_HANDLERS) {
     return false;
   }
+  BRAVE_CAN_TRACK_LAST_VISIT
 
   // Chooser based content settings will not be tracked by default.
   // Only allowlisted ones should be tracked.
