diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index c50e01e2782964fa2c9c73d0cdc39238d80f252c..13a9be94dc91243ca4806883e46187e2817d9f49 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3528,6 +3528,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
