diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index bdde3bd488b2018241835436fdaf3548dc453ec2..ecbcbe38b06cdf8f61a5442f997ff2cf24dcb26f 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3555,6 +3555,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
