diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 1cee845197313f5a6867c0809dbfabba79911512..6ad4d4a2d433d9ac42e90f0c67722731e165a267 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2482,6 +2482,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
