diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index a8b5399a644e3d6cb38e40344dd0ab65c9ee36e3..18ba73579fce14f8893bfeac52ce3e9c546c962d 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3316,6 +3316,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
